lightning/src/ln/channelmanager.rs

```diff
@@ -118,14 +119,19 @@ pub enum PendingHTLCRouting {
 		short_channel_id: u64, // This should be NonZero<u64> eventually when we bump MSRV
 	},
 	/// An HTLC paid to an invoice we generated.
```

Comment:
Small nit:

In the following line, we are saying we don't know yet if the invoice was generated by us. So adding a "supposedly" will make the comments more coherent here.

```suggestion
	/// An HTLC paid to an invoice (supposedly) generated by us.
```